tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
```

Comment:
So after reading further, there's no tool called "acrn-crashlog" so we shouldn't refer to it as a tool.  The actual tools  are "acrnprobe", "debugger", "usercrash_c", and "usercrash_s", right? And there are some Clear Linux tools used for crashlog work as well.

Reply:
You're right. No binary named as "acrn-crashlog". "acrn-crashlog" is called by a joint name for the tools "acrnprobe", "debugger", "usercrash_c", and "usercrash_s", which server collecting logs and information after each crash or event on the acrn platform. So we called it as "acrn-crashlog".